vividus-to-zephyr-exporter/src/main/java/org/vividus/zephyr/exporter/ZephyrExporter.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 2019-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.zephyr.exporter;
+
+import static org.apache.commons.lang3.Validate.notEmpty;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.TreeSet;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.MapperFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.vividus.jira.JiraFacade;
+import org.vividus.jira.model.JiraEntity;
+import org.vividus.zephyr.configuration.ZephyrConfiguration;
+import org.vividus.zephyr.configuration.ZephyrExporterProperties;
+import org.vividus.zephyr.configuration.ZephyrFileVisitor;
+import org.vividus.zephyr.databind.TestCaseDeserializer;
+import org.vividus.zephyr.facade.IZephyrFacade;
+import org.vividus.zephyr.facade.ZephyrFacade;
+import org.vividus.zephyr.model.ExecutionStatus;
+import org.vividus.zephyr.model.TestCase;
+import org.vividus.zephyr.model.TestCaseStatus;
+import org.vividus.zephyr.model.ZephyrExecution;
+
+public class ZephyrExporter
+{
+    private static final Logger LOGGER = LoggerFactory.getLogger(ZephyrExporter.class);
+
+    private final JiraFacade jiraFacade;
+    private IZephyrFacade zephyrFacade;
+    private ZephyrExporterProperties zephyrExporterProperties;
+
+    public ZephyrExporter(JiraFacade jiraFacade, ZephyrFacade zephyrFacade,
+            ZephyrExporterProperties zephyrExporterProperties) throws IOException
+    {
+        this.jiraFacade = jiraFacade;
+        this.zephyrFacade = zephyrFacade;
+        this.zephyrExporterProperties = zephyrExporterProperties;
+    }
+
+    public void exportResults() throws IOException
+    {
+        ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
I propose to move this parsing logic into separate class, this will make code cleaner and simplier

Reply:
```java
public void exportResults()
{
List<TestCase> testCasesForImporting = someParser.createTestCases(...);
ZephyrConfiguration configuration = zephyrFacade.prepareConfiguration();
for (TestCase testCase : testCasesForImporting)
        {
            createNewTestExecution(testCase, configuration, objectMapper);
        }
}
```